addons/packages/antrea/0.11.3/bundle/config/overlay/antrea_overlay.yaml

```diff
@@ -36,7 +36,7 @@ featureGates:
 #! 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run
 #! OVS in userspace mode. Userspace mode requires the tun device driver to be available.
 #@ if values.infraProvider == "docker":
-ovsDatapathType: netdev
+ovsDatapathType: system
```

Comment:
should we remove `if` and comment the configuration so it will defaults to system?

Reply:
Yeah, thats may be better. I can update and test this.

Reply:
> Could you add `patchForAntrea` back in this PR as we discussed before? Otherwise the new tanzu CLI may not work for older antrea packages as they require that patch. I think it's harmless to have that patch for newer antrea packages.

I think newer tanzu cli will always use the updated TKR? So we dont need this anymore? As long as we fix all the supported antrea versions.

Reply:
If that's the case, then it's fine to remove `patchForAntrea`. I thought tanzu CLI and package are independent.